frontend/src/components/Catalog.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import {
+  Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  CardSubtitle,
+  Button
+} from 'reactstrap';
+import { Container, Row, Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+class Catalog extends Component {
+  state = {
+    page: 1
+  };
+
+  getFirstRow = () => {
+    var page = this.state.page;
+    var start = (page - 1) * 6;
```

Comment:
nit: use let instead of var 
var creates a global variable, while let is local

Reply:
applies for other variables below too

Reply:
TIL